packages/dashmate/README.md

```diff
@@ -47,6 +47,7 @@ $ npm install -g dashmate
 
 ## Update
 
+The `update` command is used to quickly get the latest patches for docker image. It is required to restart the node after the update is complete.
```

Comment:
```suggestion
The `update` command is used to quickly get the latest patches for docker images. It is necessary to restart the node after the update is complete.
```

Reply:
Hey, I've done some changes to the description